### ISSUE TITLE
How to finetune the model

### ISSUE THREAD

**Issue by kyzira** [DepthAnything/Video-Depth-Anything#75]
How can i finetune the model to be better suited for my usecase?

**Comment by xpyqiubai** [comment]
I have the same problem too, because I hope to use my own data fine-tuning to a metric-depth video model.

**Comment by czguo** [comment]
> I have the same problem too, because I hope to use my own data fine-tuning to a metric-depth video model.

+1
